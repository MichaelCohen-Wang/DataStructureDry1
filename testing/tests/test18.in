add_ship 1832481 12
get_cannons 1832481
add_pirate 344477 1832481 21
add_pirate 303577 1832481 21
get_treasure 344477
add_ship 38403 4
add_ship 2195059 8
add_pirate 1927644 2195059 30
add_ship 252368 10
add_ship 1301688 29
remove_pirate 303577
remove_pirate 1927644
add_ship 1510389 15
add_pirate 442217 1301688 9
update_pirate_treasure 442217 20
add_ship 1082645 3
get_cannons 2195059
add_pirate 2369870 1832481 27
add_ship 1553670 25
add_ship 938003 26
remove_ship 1553670
add_ship 2447844 30
add_pirate 505188 1832481 27
add_pirate 880389 1082645 30
add_ship 1523140 8
get_cannons 2195059
add_pirate 1470600 1082645 1
add_pirate 21567 2447844 3
add_ship 1103059 8
remove_ship 1553670
remove_ship 1510389
add_ship 2564 23