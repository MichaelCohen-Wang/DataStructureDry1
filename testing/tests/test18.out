add_ship: SUCCESS
get_cannons: SUCCESS, 12
add_pirate: SUCCESS
add_pirate: SUCCESS
get_treasure: SUCCESS, 21
add_ship: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 8
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 8
add_pirate: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
remove_ship: SUCCESS
add_ship: SUCCESS
